src/Common/Core/Test/Fakes/Shell/TestFileDialog.cs

```diff
@@ -10,11 +11,18 @@ public class TestFileDialog : IFileDialog {
         public string OpenFilePath { get; set; }
         public string BrowseDirectoryPath { get; set; }
         public string SaveFilePath { get; set; }
+        public ExportImageParameters ExportImageParameters { get; set; }
+        public ExportPdfParameters ExportPdfParameters { get; set; }
 
         public string ShowOpenFileDialog(string filter, string initialPath = null, string title = null) => OpenFilePath;
 
         public string ShowBrowseDirectoryDialog(string initialPath = null, string title = null) => BrowseDirectoryPath;
 
         public string ShowSaveFileDialog(string filter, string initialPath = null, string title = null) => SaveFilePath;
+
+        public ExportImageParameters ShowExportImageDialog(ExportArguments imageArguements, string filter, string initialPath = null, string title = null) => ExportImageParameters;
```

Comment:
Do you have a recommended place for it? It has to be in a place where it's easily faked for export image/pdf ui tests, which shouldn't bring up the actual dialog.